src/types/index.ts

```diff
@@ -139,23 +103,49 @@ export interface ICellIncluded extends ITemplateIncluded {
   }
 }
 
-export interface IDashboardTemplate extends ITemplate {
-  content: {
-    data: IDashboardTemplateData
-    included?: IDashboardTemplateIncluded[]
+export interface ILabelIncluded extends ITemplateIncluded {
+  type: TemplateType.Label
+  attributes: {name: string; properties: ILabelProperties}
```

Comment:
is there a reason that attributes for ILabelIncluded is not Label in the way that it is Cell for IcellIncluded and View?

Reply:
fixed it thanks!